### ISSUE TITLE
Swiping application pages

### ISSUE THREAD

**Issue by jcperil** [Iamlooker/Droid-ify#171]
Ability to swipe left/right to jump from one application page to another in the same category (Explore, Installed, Updates).

**Comment by gallegonovato** [comment]
Good afternoon @jcperil

What you indicate can already be done.

You can swipe right from browse. And it will take you to Installed.

And the other way around also works.

Best regards

**Comment by jcperil** [comment]
I think you misunderstood. I am not swiping from Explore category to Installed category or vice versa. Let's fix a category, say the Explore section. Click on any app on the explore section to open its page. I would then like to be able to swipe left/right to open page of the next app in Explore category.

**Comment by gallegonovato** [comment]
Good afternoon @jcperil

First of all let me apologize for taking so long to get back to you. But here it is very bad weather and we still don't have an internet connection, which does not fail because of this. Let's see if it gets better as they are now putting the cables underground.

And also apologize for not understanding at first what you were saying.

I am not very fond of gestures. And maybe what you say can lead to install some application after an accidental press, sliding or swiping between applications.

But it's not up to me, let's see what @Iamlooker thinks and if he can implement it in a next version of Droid-ify.

Best regards and thanks for your suggestion :)

**Comment by Rend0e** [comment]
Agreed, I think putting 'Explore' tab in the middle and setting it as the default one when you open the app would be better than the defaults.  It's the most used tab and would result in only one swipe gesture to access the 'Updates' tabs. 'Updates' tab is used more than 'installed' tab so swiping once to access it would be nicer than having to swipe twice each time you open the app.

**Comment by uli-on** [comment]
@Rend0e As far as I understood the initial proposal I think yours is one more misunderstanding of jcperil's proposal. S/He suggests a swiping gesture from single app's details page to single app's details page, restricted to the currently viewed tab.

**Comment by jcperil** [comment]
@uli-on is right. I am requesting support for swiping between application pages. Although it might be hard to implement because swiping also helps us see screenshots on application pages.

**Comment by jcperil** [comment]
> Although it might be hard to implement because swiping also helps us see screenshots on application pages.

Perhaps not. Fritter has this feature implemented.
